src/frontend/Page/Docs.elm

```diff
@@ -43,11 +48,21 @@ type alias Model =
   , latest : Status V.Version
   , readme : Status String
   , docs : Status (List Docs.Module)
+  , manifest : Status Project.PackageInfo
```

Comment:
I used the name "manifest" because "info" (like `PackageInfo` from project-metadata-utils) was already used with the `getInfo` function, and well, it is actually a manifest.

Reply:
The name in the compiler is `outline`. I think it makes sense to try to match the terms, even though the elm-metadata-utils package has not been updated to match yet.